### PR TITLE
Calculated custom properties for the toolbar heights

### PR DESCRIPTION
**Pull request by abdonrd**
Like the `paper-checkbox`: [PolymerElements/paper-checkbox/paper-checkbox.html#L60](https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html#L60)

### CODE REVIEW

**Comment by abdonrd** [comment]
Friendly ping to @bicknellr. I see you in the last commits. :)

File: paper-toolbar.html [hunk 1]
```diff
@@ -106,14 +106,17 @@
   <template>
     <style>
       :host {
+        --calculated-paper-toolbar-height: var(--paper-toolbar-height, 64px);
+        --calculated-paper-sm-toolbar-height: var(--paper-toolbar-sm-height, 56px);
```
Comment (bicknellr) [inline paper-toolbar.html]: `--calculated-paper-sm-toolbar-height` -> `--calculated-paper-toolbar-sm-height`, to match the variable name.

Comment (abdonrd) [inline paper-toolbar.html]: Oops! Your are right! Updated! :)

**Comment by bicknellr** [comment]
LGTM, thanks! (also, [app-toolbar](https://github.com/PolymerElements/app-layout/tree/master/app-toolbar) btw)

**Pull request merged**